api/src/main/java/org/pmiops/workbench/cohortbuilder/querybuilder/DemoQueryBuilder.java

```diff
@@ -0,0 +1,80 @@
+package org.pmiops.workbench.cohortbuilder.querybuilder;
+
+import com.google.cloud.bigquery.QueryParameterValue;
+import com.google.cloud.bigquery.QueryRequest;
+import org.pmiops.workbench.model.SearchParameter;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * DemoQueryBuilder is an object that builds {@link QueryRequest}
+ * for BigQuery for the following criteria types:
+ * DEMO_GEN, DEMO_AGE, DEMO_RACE and DEMO_DEC.
```

Comment:
I don't see DEMO_RACE or DEMO_DEC below. Are those being added later?

What's DEMO_DEC short for?

Reply:
> I don't see DEMO_RACE or DEMO_DEC below. Are those being added later?
The implementation is waiting for test data I believe 

> What's DEMO_DEC short for?
deceased

Reply:
yes.. they will be added when we get the tables in bigquery. I'm not sure what needs to happen with race yet. OMOP's handling of race is much different than vandy handles it. So we need to figure out a new way. DEMO_DEC is deceased.